src/metadata.rs

```diff
@@ -27,8 +32,37 @@ pub fn get_metadata_text(root_path: &str, id: &str) -> io::Result<String> {
     fs::read_to_string(path)
 }
 
+fn get_sorted_id_string(mut ids: Vec<String>) -> String {
+    ids.sort_by(|a, b| a.cmp(b));
+    ids.join("")
+}
```

Comment:
Later on we might check to see how this does on really big lists as we might end up better with streaming the hash. Nothing to worry about now though...